### PR TITLE
Add Fordermittel Digitale Medien Blog

### PR DESCRIPTION
**Pull request by lennartdefacto**
- [x] Create blog
- [x] Add Text
- [x] add social images
- [x] feature blog on homepage
- [x] add option in text to mail colleague A. Dijkslag

### CODE REVIEW

File: source/blog/de/2018-09-27-fordermittel-fur-digitale-medien-in-der-beruflichen-bildung-im-gesundheitswesen.html.markdown [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Alle Institutionen aus den Bereichen Forschung und Entwicklung im Gesundheitsbereich sowie Berufliche Aus-, Fort- und Weiterbildung in Gesundheitsberufen sind antragsberechtigt. Das Antragsverfahren ist zweistufig angelegt. Die erste Stufe des Verfahrens hat als Einsendeschluss den 19. Oktober 2018.
+
+Möchten Sie einen Antrag für Fördermittel stellen? Vordrucke für Förderanträge, Richtlinien, Merkblätter, Hinweise und Nebenbestimmungen können auf folgender [Webseite abgerufen werden.](https://foerderportal.bund.de/easy/easy_index.php?auswahl=easy_formulare&formularschrank=bmbf){:target="_blank"}{:rel="noopener"}{:title="Förder Portal."}
+
+Haben Sie weitere Fragen zu dieser Ausschreibung? Nehmen Sie Kontakt mit unserem Customer Relationshop Manager <strong><a href="a.dijkslag@defacto.nl">Albert Dijkslag</a></strong> auf.
```
Comment (RonaldDijkstra) [inline source/blog/de/2018-09-27-fordermittel-fur-digitale-medien-in-der-beruflichen-bildung-im-gesundheitswesen.html.markdown]: Markitdown: 

`[Albert Dijkslag](mailto:a.dijkslag@defacto.nl)`

File: source/blog/de/2018-09-27-fordermittel-fur-digitale-medien-in-der-beruflichen-bildung-im-gesundheitswesen.html.markdown [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: "Fördermittel für Digitale Medien in der beruflichen Bildung im Gesundheitswesen"
+date: 2018-09-27 08:00 CEST
+tags: Ausbildung, Fortbildung, Weiterbildung, Gesundheitswesen, Digitalisierung Krankenhaus, DigiMed
```
Comment (RonaldDijkstra) [inline source/blog/de/2018-09-27-fordermittel-fur-digitale-medien-in-der-beruflichen-bildung-im-gesundheitswesen.html.markdown]: I think this is a bit too much taggies

File: data/news_de.yml [hunk 3]
```diff
@@ -10,8 +10,8 @@
   link: "/jobs/projektleiter-consultant-softwareimplementierung/"
   alt: "Werde Teil unseres Teams und arbeite aktiv an unserem Motto 'Developing People' mit!"
 
-- title: "Fort- und Weiterbildung für Medizinische Fachangestellte"
-  text: "Immer mehr Kliniken beschäftigen Medizinische Fachangestellte und fördern deren weitere Qualifikation, um in Zeiten des Fachkräftemangels Mediziner und Pflegekräfte gezielt zu entlasten."
-  image: "news/mfa-blog-2.jpg"
```
Comment (RonaldDijkstra) [inline data/news_de.yml]: Remove image from source please

File: source/blog/de/2018-09-27-fordermittel-fur-digitale-medien-in-der-beruflichen-bildung-im-gesundheitswesen.html.markdown (hunk 1 above)
Comment (lennartdefacto) [reply source/blog/de/2018-09-27-fordermittel-fur-digitale-medien-in-der-beruflichen-bildung-im-gesundheitswesen.html.markdown]: changed.

**Pull request merged**